<launch>

  <arg name="do_head_reset" default="true"/> 

  <node pkg="arm_head_control" name="HeadInterface" type="arm_head_control" output="screen">
    <param name="do_head_reset" value="$(arg do_head_reset)"/>
  </node>

</launch>